comp/netflow/forwarder/component.go

```diff
@@ -0,0 +1,54 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+// Package forwarder exposes the event platform forwarder for netflow.
+package forwarder
+
+import (
+	"testing"
+
+	"go.uber.org/fx"
+
+	"github.com/DataDog/datadog-agent/pkg/aggregator"
+	"github.com/DataDog/datadog-agent/pkg/epforwarder"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+	"github.com/golang/mock/gomock"
+)
+
+// team: network-device-monitoring
+
+// Component is the component type.
+type Component interface {
+	epforwarder.EventPlatformForwarder
+}
+
+// Module defines the fx options for this component.
+var Module = fxutil.Component(
+	fx.Provide(getForwarder),
+)
+
+func getForwarder(agg aggregator.DemultiplexerWithAggregator) (Component, error) {
+	return agg.GetEventPlatformForwarder()
+}
+
+// MockComponent is the type for mock components.
+type MockComponent interface {
+	Component
+	EXPECT() *epforwarder.MockEventPlatformForwarderMockRecorder
```

Comment:
Is there a reason to not call this function `Expect`?

Reply:
I didn't choose that name, it's part of https://github.com/DataDog/datadog-agent/blob/main/pkg/epforwarder/epforwarder_mockgen.go#L33

It's generated by gomock; I think they use the name EXPECT because it's very unlikely that a mocked class will have a method with that name already?